easylist_adult/adult_specific_block_popup.txt

```diff
@@ -1,5 +1,5 @@
 .com./$popup,domain=pornhub.com|pornhubthbh7ap3u.onion
-|http*://*?$popup,third-party,domain=pornhub.com|pornhubthbh7ap3u.onion|redtube.com|tube8.com|youporn.com|youporngay.com
+|http*://*?$popup,third-party,domain=pornhub.com|pornhubthbh7ap3u.onion|redtube.com|tube8.com|watchomovies.pro|youporn.com|youporngay.com
```

Comment:
`pornhubthbh7ap3u.onion` is a outdate v2 onion address which should be removed or updated